docs/conf/portland/2024/news/announcing-cfp.rst

```diff
@@ -0,0 +1,57 @@
+:template: {{year}}/generic.html
+
+.. post:: Nov 06, 2023
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+Announcing the Call for Proposals
+=======================================
+
+We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, starting earlier than last year, as the conference is also a little earlier in the year.
+
+Speaking at Write the Docs is a fantastic way to share your ideas with our inspiring community of documentarians.
+If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+We're looking for talk proposals to be delivered in-person, on-stage in Portland.
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.
+
+The Call for Proposals will be open until **11:59 PM {{cfp.ends}} PST**.
+
+(If you'd prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for `Atlantic_news_2024`_ held online in a timezone as widely accessible as possible).
+
+.. _Atlantic_news_2024:
+
+Write the Docs Atlantic, online date TBD
+----------------------------------------
+
+If you want to speak at Write the Docs, but can't or don't want to attend an in-person event, keep an eye out for Write the Docs Atlantic, an online conference coming later in 2024.
+
+This conference will be entirely virtual, in a timezone that works for as many people as possible, so you can present or attend from anywhere.
+
+Tickets
+-------
+
+Tickets will officially go on sale early **2024**.
+
+As always, we keep ticket prices low so that the event is accessible to the widest range of people possible.
+However, if you can't afford the tickets and still wish to attend, we will launch our grants program soon after opening ticket sales.
+
+Stay Updated
```

Comment:
We should use this announcement to ship our sponsorship program, which is likely the other thing we should be highlighting. Will work on that this week.

Reply:
Yep, lemme know if you need a hand. Will just need to update the blog post date when we do put it out.